xml/security_ldap.xml

```diff
@@ -398,266 +399,195 @@ objectclass (2.16.840.1.113730.3.2.333 NAME 'nsPerson' <co xml:id="co-ldap-schem
-         Set the <varname>root_password</varname> to the password for
-         the directory server &rootuser; user. The password is used for LDAP
-         server administration only.
+         Create a <varname>root_password</varname> for
+         the directory server &rootuser; user, <literal>dirsrv</literal>. 
+         The <literal>dirsrv</literal> user has no login or shell privileges, 
+         and is used for LDAP server administration only.
+        </para>
+       </callout>
+       <callout arearefs="co-ldap-389-ds-cert">
+        <para>
+         Create self-signed server certificates in 
+         <filename>/etc/dirsrv</filename>.
         </para>
        </callout>
+       <callout arearefs="co-ldap-389-ds-sample-entries">
+        <para>
+         Populate the new instance with sample user and group entries.
+        </para>
+       </callout>       
       </calloutlist>
      </example>
    </step>
    <step>
     <para>
      To create the &ds389; instance from <xref linkend="ex-ldap-389-ds-inf"
-     xrefstyle="select:label"/>, run:
+     xrefstyle="select:label"/>, run the following command:
     </para>
-    <screen>&prompt.sudo;dscreate from-file /tmp/instance.inf</screen>
+    <screen>&prompt.sudo;dscreate -v from-file 389ds-test.inf | tee 389ds-test-output.txt</screen>
     <para>
-     This creates a working LDAP server.
+     This shows all activity during the instance creation, stores all the messages in
+     <filename>389ds-test-output.txt</filename>, and creates a working 
+     LDAP server in about a minute. The verbose output contains a lot of useful
+     information. If you don't want to save it then delete the 
+     <literal>| tee 389ds-test-output.txt</literal> portion of the command.
     </para>
    </step>
    <step>
     <para>
-     If <command>dscreate</command> should fail, the messages will tell you why.
-     For more details, repeat the command with the <option>-v</option> option:
+     If the <command>dscreate</command> command should fail, the messages will 
+     tell you why. After correcting any issues, re-run the command.    
     </para>
-    <screen>&prompt.sudo;dscreate -v from-file /tmp/instance.inf</screen>
    </step>
    <step>
     <para>
-     Check the status of the server with:
+     A successful installation reports "<literal>Completed installation for 
+     389ds-test</literal>". Check the status of your new server:
     </para>
-    <screen>&prompt.sudo;dsctl localhost status
-instance 'Localhost' is running</screen>
+    <screen>&prompt.sudo;dsctl 389ds-test status
+Instance "389ds-test" is running</screen>
    </step>
    <step>
     <para>
-     In case you want to delete the instance later on:
+     The following command cleanly removes the instance. You may reliably create 
+     and remove instances as much as you want:
     </para>
-    <screen>&prompt.sudo;dsctl localhost remove --do-it</screen>
+    <screen>&prompt.sudo;dsctl 389ds-test remove --do-it</screen>
```

Comment:
Honestly, was this at one point was --Isolemlysweariamuptonogood, but I think --yolo would also be cool too. 

But the real comment here is in doc's I "prefer" not to add the "--do-it". That way if someone copy pastes they hit the dry run mode *first*. It then prompts "okay add --do-it to really do this". So generally if possible, please don't add the --do-it in command examples *OR* at least seperate it from the copy-paste examples so that it can't yield a copy-paste mistake.

Reply:
ok I will add the dry-run command.